ecl/hql/hqlexpr.hpp

```diff
@@ -1287,10 +1287,8 @@ extern HQL_API IHqlExpression *createValue(node_operator op, ITypeInfo * type);
 extern HQL_API IHqlExpression *createValue(node_operator op, ITypeInfo * type, IHqlExpression *p1);
 extern HQL_API IHqlExpression *createValue(node_operator op, ITypeInfo * type, IHqlExpression *p1, IHqlExpression *p2);
 extern HQL_API IHqlExpression *createValue(node_operator op, ITypeInfo * type, IHqlExpression *p1, IHqlExpression *p2, IHqlExpression *p3);
```

Comment:
In some places there is createValue(op, type, expr1,expr2) and in other places createValue(op, type, {expr1,expr2}).  Without examining the code, it was not clear that these do exactly the same thing. May be worth putting a comment here and adding a note to remove the older form.